Source/Train-Railway/Station.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace Train_Railway
+{
+    class Station
+    {
+        public int ID { get; set; }
+        public string StationName { get; set; }
+        public bool EndStation { get; set; }
+        public bool IsAvaliable { get; set; } = true;
+
+        public List<Passenger> Travelers = new List<Passenger>();
```

Comment:
I would have called this list Passengers

Reply:
I agree, Passengers is a better name.
Unless it's a special kind of passengers